blocks/top-table-list-block/features/top-table-list/default.jsx

```diff
@@ -458,6 +458,8 @@ TopTableListWrapper.propTypes = {
   }),
 };
 
+TopTableListWrapper.lazy = true;
```

Comment:
this is new?

Reply:
My bad, I was testing the lazy loading stuff - removed